frontend/src/utils/helpers/moduleUtils.js

```diff
@@ -33,6 +33,15 @@ function isModuleEnabled(module) {
   return getModuleActiveStatus(module);
 }
 
+function isAnyFeatureModuleEnabled() {
```

Comment:
I'm a little unsure about this method.. We basically have no nice way to tell if a module is a feature module or not. If we want to avoid this hardcoded if.. We could add a flag into the config.json that a module is considered a feature module... 

Or perhaps make a new component like EnvironmentFeatures.js but with Toggles? And loop through the array? Let me know what you prefer.

Reply:
To this issue we also have the challenge of managing dependencies. At the moment they are hardcoded in the `isModuleEnabled` function. Maybe we can think of a generic way of defining some module properties in the frontend as we do in the `__init__` files for backend. I spoke about this with Mo as one of the future improvements for frontend. For the moment i think it is good